package.de-de.json

```diff
@@ -0,0 +1,17 @@
+{
+  "deno.displayName": "Deno",
+  "deno.description": "Deno unterstützung für VSCode",
+  "deno.command.enable": "Aktiviere Deno",
+  "deno.command.disable": "Deaktiviere Deno",
+  "deno.command.restart_server": "Starte den Deno Language Server neu",
+  "deno.config.enabled": "Steuert, ob Deno aktiviert ist oder nicht.\n\n**In der globalen Konfiguration nicht empfohlen**",
+  "deno.config.dtsFilepaths": "Der Dateipfad der Deno TypeScript-Deklarationsdatei (.d.ts).\n\nDies kann ein Pfad sein, der relativ zum Projektpfad ist, oder ein absoluter Pfad.\n\n**Wird in der globalen Konfiguration nicht empfohlen, ausser wenn Sie einen absoluten Pfad verwenden.**",
+  "deno.config.import_map": "Der Dateipfad einer import map.\n\nDies kann ein Pfad sein, der relativ zum Projektpfad ist, oder ein absoluter Pfad.\n\n**In der globalen Konfiguration nicht empfohlen**",
+  "deno.initializing": "Deno Language Server wird initialisiert ...",
+  "err.not_install_deno": "Konnte `deno` in Ihrem $PATH nicht finden. Bitte installieren Sie `deno` und starten Sie die Erweiterung neu.",
+  "diagnostic.fix.create_module": "Erstellen das Modul",
+  "diagnostic.fix.fetch_module": "Download das Modul",
+  "diagnostic.fix.lock_std_version": "Sperren Sie das Modul auf die aktuelle Deno-Version",
```

Comment:
Can you remove this as it is no longer used

- `diagnostic.fix.lock_std_version `
- `diagnostic.report.lock_std_version `

Reply:
I must say, I really liked these... oh well.